project/build.sbt

```diff
@@ -20,11 +20,3 @@ addSbtPlugin("com.eed3si9n"       % "sbt-dirty-money"   % "0.2.0")
 addSbtPlugin("org.foundweekends"  % "sbt-bintray"       % "0.5.4")
 addSbtPlugin("com.jsuereth"       % "sbt-pgp"           % "2.0.0")
 addSbtPlugin("com.typesafe"       % "sbt-mima-plugin"   % "0.6.1")
-
-scalacOptions ++= Seq(
-  "-deprecation",
-  "-unchecked",
-  "-feature",
-  "-encoding",
-  "utf8"
-)
```

Comment:
This should be kept. These settings apply to the build itself. If you remove them, we won't see warnings happening in the build itself anymore.

Reply:
Restored as suggested.